### PR TITLE
[12.0][MIG] sql_request_abstract

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
Migration of module sql_request_abstract to v12

### CODE REVIEW

**Review by legalsylvain — APPROVED** [review]
code review of the diff from 11.0 (last commit) / No test.
LGTM and simple.

Thanks for porting this module ! :+1:

File: sql_request_abstract/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Inherit the model:
+
+    from openerp import models
```
Comment (bealdav) [inline sql_request_abstract/readme/USAGE.rst]: from odoo import models

**Review by bealdav — APPROVED** [review]

File: sql_request_abstract/readme/USAGE.rst (hunk 1 above)
Comment (bealdav) [reply sql_request_abstract/readme/USAGE.rst]: LGTM, only old string here, thanks

**Review by MiquelRForgeFlow — APPROVED** [review]
Please, squash last commit

**Comment by HviorForgeFlow** [comment]
Rebase needed?

**Review by MiquelRForgeFlow — APPROVED** [review]
Normally when you squash, you rebase also at the same time haha

**Review by CasVissers-360ERP — APPROVED** [review]
Functional test OK. LGTM.

**Comment by HviorForgeFlow** [comment]
> Normally when you squash, you rebase also at the same time haha

I see more than one commit here I would do a rebase to OCA remote 11.0

@ageficent

**Comment by MiquelRForgeFlow** [comment]
@hveficent, @ageficent shouldn't squash commit history, only his commits.

**Pull request merged**